sqld/src/query.rs

```diff
@@ -130,6 +136,21 @@ pub enum Value {
     Blob(Vec<u8>),
 }
 
+impl Value {
+    /// Approx. of the size in bytes required to store the value as json
+    fn size_hint(&self) -> usize {
```

Comment:
This function only estimates size of the value itself, but the JSON representation also includes the type (i.e., integers are represented as `{"type":"integer","value":"123"}`. I think that we can improve the estimate by adding 30 to the value returned from the `match`.